# Message describing the sensor joint currents
Header header
string[] names
float32[] currents